src/main/java/com/ericsson/ei/jmespath/IncompletePathFilterFunction.java

```diff
@@ -56,15 +67,15 @@ public IncompletePathFilterFunction() {
      * Flatten an object and creates a list with the parts of search key. Returns array that contains filtered values.
      */
     private ArrayList<String> filterObjectWithIncompletePath(String object, String key) {
-        Map<String, Object> flattJson = JsonFlattener.flattenAsMap(object);
+        Map<String, Object> flattJson = flatten(object);
         ArrayList<String> resultArray = new ArrayList<String>();
         List<String> keyParts = Arrays.asList(key.split("\\."));
-        looptThroughAllPaths(resultArray, flattJson, keyParts);
+        loopThroughAllPaths(resultArray, flattJson, keyParts);
```

Comment:
I think loopThroughAllPaths can be a bit confusing since it also updates the resultArray without any return. I think the variable name should make this clear when used. Otherwise i have no further comments :)

Reply:
Done